src/config/queue.ts

```diff
@@ -84,7 +90,45 @@ async function initializePatientQueue(conn: Connection) {
                     `Message Queue: Received ${testPayload.length} test data`
                 );
                 if (testPayload.length > 0) {
-                    // TODO: Preprocess and load to database
+                    //TODO: Preprocess and load to database
+                    var newZipMap = new Map<Number, Number>();
+                    zipList = [];
+                    var now = new Date()
+                    //this is so scuffed
```

Comment:
Lol, to avoid any edge cases and to make life easier for you, you can install and use a library called `lodash` which handles a chunk of the object and array processing for you in Javascript!

Reply:
Ooh. I'll have to look into it, then! I don't mind writing out functions like this but it's not very pleasing to the eyes.